src/components/Tabs/Tabs.tsx

```diff
@@ -14,8 +14,9 @@ import { getWidthFromRef } from "./utils"
 
 /* The tab being dragged is appended to the body, hence it's styles need to be at global scope*/
 const GlobalStyle = createGlobalStyle`
-.knitui-tabs-helper {
+.knitui-tabs-helper:not(#knit-active-tab) {
```

Comment:
sortable-helper will always have both `#knit-active-tab` & `.knitui-tabs-helper` class

Reply:
Not in the case where an inactive tab is being dragged. @prasook-jain

Reply:
Got it.. :+1: